docs/cloud/faq/roadmap.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Migrating to Cloud
+
+### What is the ETA on the tool being developed to assist in migration of existing customer data?
+
+We have a replication tool, which is available now. It has certain limitations, especially with performance. Get in touch, so we can help you to analyse your setup and requirements, before we can recommend using the replication tool.
```

Comment:
@alexeyzimarev can we link the replicator page?

Reply:
No, we don't have a decision to make it public.